src/js/components/Box/Box.js

```diff
@@ -201,6 +212,14 @@ const Box = forwardRef(
         elevationProp={elevation}
         fillProp={fill}
         focus={focus}
+        gap={
+          boxOptions?.cssGap &&
+          gap &&
+          gap !== 'none' &&
+          border !== 'between' &&
+          border?.side !== 'between' &&
+          gap
```

Comment:
It looks like we check for gap twice
```suggestion
          boxOptions?.cssGap &&
          gap &&
          gap !== 'none' &&
          border !== 'between' &&
          border?.side !== 'between'
```

Reply:
We want to return the actual `gap` value if all prior checks are true.

Reply:
Oops, that makes sense